### ISSUE TITLE
System Update : Namespace Collision for Plugins

### ISSUE THREAD

**Issue by magnessjo** [craftcms/cms#3457]
### Description

This is a very low priority issue.  

I created a quick plugin today for a client and I was being lazy can called it Contact Form (contact-form).  I generally namespace my plugins with my initials (even though they are primary for internal use only), but today I was lazy.  

Upon completion of my plugin, I noticed a bunch of updates in the utilities.  I had nine updates for a contact form plugin that I didn’t create or download.  It appears to be the contact form for CraftCMS 2.*.*.  that was created by Pixel&Tonic.  Please see screenshot.  

I know this issue could be very easily avoided by name spacing your plugin correctly, but I thought I should report so it is known for anyone else and for a possible explanation of what could be occurring.  

### Steps to reproduce

1. Create a plugin called Contact Form

![screen shot 2018-11-08 at 4 13 18 pm](https://user-images.githubusercontent.com/856595/48227977-e35afa00-e371-11e8-9823-d3981a430f60.png)

**Comment by brandonkelly** [comment]
Yeah, I think it might make more sense for us to identify plugins by their package names rather than handles in Craft 4.
